src/GeekLearning.Email/IEmailProvider.cs

```diff
@@ -1,11 +1,18 @@
-﻿namespace GeekLearning.Email
+namespace GeekLearning.Email
 {
     using System.Collections.Generic;
     using System.Threading.Tasks;
-
+    using MimeKit;
+    
     public interface IEmailProvider
     {
-        Task SendEmailAsync(IEmailAddress from, IEnumerable<IEmailAddress> recipients, string subject, string bodyText, string bodyHtml);
+        Task SendEmailAsync(
```

Comment:
`IEmailProvider` should probably use `AddressTarget`

Reply:
My idea was to be able to define a complete email transaction in one call. You might want to send to two (or more) different addresses, Cc: another, and Bcc: yet another, and also have a ReplyTo: for the message that is different that the From: address. (some hosting services required the From address to match the authentication user name, which might not be the same as the 'sender' of the immediate message). Using an attribute parameter for the email address implies a single target transaction. Slower if the app wants to send to several targets.